semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/span/SpanStore.java

```diff
@@ -0,0 +1,118 @@
+package org.hypertrace.semantic.convention.utils.span;
+
+import com.typesafe.config.Config;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+import org.hypertrace.core.datamodel.AttributeValue;
+import org.hypertrace.core.datamodel.Attributes;
+import org.hypertrace.core.datamodel.Event;
+import org.hypertrace.core.spannormalizer.SpanIdentity;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import redis.clients.jedis.DefaultJedisClientConfig;
+import redis.clients.jedis.Jedis;
+
+public class SpanStore {
```

Comment:
please move span store to a separate module - it doesn't belong as a util, especially one as widely consumed as this.

Reply:
I have created this PR for some quick testing. I don't plan to merge it. Will close this PR post testing.